### PR TITLE
chore: drop node8, update README.md

### PR DESCRIPTION
**Pull request by aabmass**
Dropping support for node8. Node8 is EOL and hasn't seen a release since last December (almost a year). It is blocking upgrading many packages because their new releases don't support node8. This includes google libraries, e.g. `google-auth-library` which has a security vulnerability (#186) but the fix has only been released for node10+.

Updated the CI to run for versions 10, 12, 14, and 15, which are the [officially supported Node.js versions](https://nodejs.org/en/about/releases/).

The OTel JS SIG is keeping support for node8 in the core packages, because they have very few dependencies. For exporters and instrumentations, which generally have more deps, they will not be uniformly supporting node8.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # OpenTelemetry Operations Exporters for JavaScript
 
+The packages in this repository support Node.js version 10 and above.
```
Comment (nilebox) [inline README.md:3]: Why not 9?
Based on the Circle CI config above, we never checked support for Node 9 and 11 (not sure what are the reasons for that).
Do we need to mention not supporting Node 11 then? Or do we need to add a missing coverage to Circle CI config?

Comment (aabmass) [reply README.md:3]: Even numbered releases are LTS, so node9 and node11 have been deprecated for a while ([npm releases info](https://nodejs.org/en/about/releases/)):
> Major Node.js versions enter Current release status for six months, which gives library authors time to add support for them. After six months, odd-numbered releases (9, 11, etc.) become unsupported, and even-numbered releases (10, 12, etc.) move to Active LTS status and are ready for general use. LTS release status is "long-term support", which typically guarantees that critical bugs will be fixed for a total of 30 months. Production applications should only use Active LTS or Maintenance LTS releases.

image

Comment (aabmass) [reply README.md]: Updated :)

**Review by nilebox — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nilebox) [reply README.md]: Thanks!

**Pull request merged**